src/xrpld/app/misc/WamrVM.cpp

```diff
@@ -640,6 +640,62 @@ WamrEngine::call(
     return call<NR>(func, in, p.data(), p.size(), std::forward<Types>(args)...);
 }
 
+Expected<int32_t, NotTEC>
+WamrEngine::preflight(
+    wbytes const& wasmCode,
+    std::string_view funcName,
+    std::vector<WasmImportFunc> const& imports,
+    std::vector<WasmParam> const& params)
+{
+    try
+    {
+        return preflightHlp(wasmCode, funcName, imports, params);
+    }
+    catch (std::exception const& e)
+    {
+        std::cerr << "WAMR: preflight exception: " << e.what() << std::endl;
```

Comment:
It is better to use rippled logs

Reply:
Waiting on #5395 - will switch all these error messages to rippled logs after that